core/experiments/manager.php

```diff
@@ -439,19 +439,6 @@ private function add_default_features() {
 			'release_status' => self::RELEASE_STATUS_ALPHA,
 			'default' => self::STATE_ACTIVE,
 		] );
-
-		// TODO: Possibly remove experiment in v3.27.0 [ED-15717].
```

Comment:
@rami-elementor I believe it is safe to remove this experiment, as it was hidden and set to immutable for 2 versions already.

Reply:
I agree